src/analytics.ts

```diff
@@ -58,8 +58,8 @@ export function createAnalyticsInstance(options?: Options) {
                         device_type: growthbookOptions?.attributes.device_type,
                     }),
                     ...(growthbookOptions?.attributes?.url && { url: growthbookOptions?.attributes.url }),
-                    ...(growthbookOptions?.attributes?.loggedIn && {
-                        loggedIn: growthbookOptions?.attributes.loggedIn,
+                    ...(growthbookOptions?.attributes && {
+                        loggedIn: growthbookOptions?.attributes?.loggedIn,
```

Comment:
the question is if the value is undefined this gonna pass undefined to GB which is not the intention.
There would be two option to fix this
1.
```suggestion
                    ...(growthbookOptions?.attributes !== undefined && {
                        loggedIn: growthbookOptions?.attributes?.loggedIn,
```
2. 
```suggestion
                    ...(growthbookOptions?.attributes && {
                        loggedIn: !!growthbookOptions?.attributes?.loggedIn,
```